tests/ui/pages/data_entity/overview.page.ts

```diff
@@ -11,9 +12,24 @@ const SELECTORS = {
   addTagButton: `.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add Tags')`,
   inputTagName: `[role="combobox"]`,
   saveButton: `[type="submit"]`,
+  addCustomDescription: `.MuiButtonBase-root.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeMedium:has-text('Add description')`,
+  customDescriptionInput: `.w-md-editor-text-input`,
+  saveDescriptionButton: `button:text-is('Save')`,
+  addCustomMetadata: `.MuiButtonBase-root.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeMedium:has-text('Add metadata')`,
```

Comment:
could you please review all locators in your PR and get rid of technical classes from them